employees/common/exports.py

```diff
@@ -216,3 +224,58 @@ def _get_report_date_and_daily_hours(self, current_report: Report, reports_subse
 
 def convert_markdown_html_to_text(html: str) -> str:
     return "".join(BeautifulSoup(html, features="html.parser").findAll(text=True))
+
+
+def save_work_book_as_csv(writer: csv.DictWriter, work_book: Workbook) -> None:
+    sheet = work_book.get_active_sheet()
+    is_last_row = False
+    total_hours = timezone.timedelta()
+
+    for row_number, row in enumerate(sheet.rows, start=1):
+        if row_number == 1:
+            continue
+
+        # Check if is last row by comparing first cell value to `TOTAL`.
+        if isinstance(row[0].value, str) and row[0].value == constants.TOTAL.value:
+            is_last_row = True
+
+        next_row = []
+        for cell_number, cell in enumerate(row, start=1):
+            if isinstance(cell.value, str) and cell.value.lower().startswith("=timevalue"):
+                hours_as_string = cell.value[len('=timevalue("') : -len('")')]
+                next_row.append(hours_as_string)
+
+                # If contains valid hours value, add it to total hours.
+                if (
+                    not is_last_row
+                    and row_number > 2
+                    and cell_number
+                    == constants.HEADERS_TO_COLUMNS_SETTINGS_FOR_SINGLE_USER.value[
+                        constants.HOURS_HEADER_STR.value
+                    ].position
+                ):
+                    hours, minutes = convert_string_work_hours_field_to_hour_and_minutes(hours_as_string)
+                    total_hours += timezone.timedelta(hours=int(hours), minutes=int(minutes))
+            elif is_last_row and cell.data_type is TYPE_FORMULA:
+                next_row.append(str(total_hours)[:-3])
+            else:
+                next_row.append(cell.value)
+
+        writer.writerow(next_row)
+
+
+def save_work_book_as_zip_of_csv(work_book: Workbook) -> zipfile.ZipFile:
```

Comment:
Don't you return here `BytesIO` object?

Reply:
Right, ZIP was returned in previous version of the code. :+1: done